### PR TITLE
[CARBONDATA-1568] Optimize annotation of code

### PR DESCRIPTION
**Pull request by xubo245**
There are some improper places in code annotation by IDEA inspect code
We  optimize annotation of code:

-  Fix an error of Javadoc issues
- modify the expression of annotation

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/412/

**Comment by CarbonDataQA** [comment]
Build Failed with Spark 1.6, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/289/

**Comment by ravipesala** [comment]
SDV Build Fail , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/1043/

File: integration/spark/src/main/scala/org/apache/carbondata/spark/CarbonDataFrameWriter.scala [hunk 1]
```diff
@@ -59,8 +59,8 @@ class CarbonDataFrameWriter(val dataFrame: DataFrame) {
   /**
    * Firstly, saving DataFrame to CSV files
    * Secondly, load CSV files
-   * @param options
-   * @param cc
+   * @param options  CarbonOption
```
Comment (jackylk) [inline integration/spark/src/main/scala/org/apache/carbondata/spark/CarbonDataFrameWriter.scala]: add more explain for parameters.

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/451/

**Comment by CarbonDataQA** [comment]
Build Failed with Spark 1.6, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/326/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/327/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/452/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/1079/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/1080/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/494/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/498/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/1122/

**Comment by chenliang613** [comment]
LGTM

**Comment by ravipesala** [comment]
SDV Build Fail , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/1126/

**Comment by jackylk** [comment]
LGTM

**Pull request closed without merging**